capa/v29.0.0/release.yaml

```diff
@@ -108,6 +108,10 @@ spec:
     - prometheus-operator-crd
   - name: vertical-pod-autoscaler-crd
     version: 3.1.0
+  - name: observability-policies
+    version: 0.0.1
+    dependsOn:
+    - kyverno
```

Comment:
Franco changed this to `kyverno-crds` in their PR: https://github.com/giantswarm/releases/pull/1319. I'll try to get their PR merged, so I can rebase yours afterwards.